kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ConvertUtils.java

```diff
@@ -132,6 +135,47 @@ public static ProcessInstanceList convertToProcessInstanceWithVarsList(Collectio
 
         return instance;
     }
+    
+    public static ProcessInstanceCustomVarsList convertToProcessInstanceCustomVarsList(Collection<ProcessInstanceCustomDesc> instances) {
+        if (instances == null) {
+            return new ProcessInstanceCustomVarsList(new org.kie.server.api.model.instance.ProcessInstanceCustomVars[0]);
+        }
+
+        List<ProcessInstanceCustomVars> processInstances = new ArrayList<ProcessInstanceCustomVars>(instances.size());
+        for (ProcessInstanceCustomDesc pi : instances) {
+            org.kie.server.api.model.instance.ProcessInstanceCustomVars instance = convertToProcessInstanceCustomVars(pi);
+
+            instance.setVariables(pi.getVariables());
```

Comment:
why not use builder's variables() method in convertToProcessInstanceCustomVars(pi)?

Reply:
I use builder's variables inside the method convertToProcessInstanceCustomVars(pi) as the rest of mappers we've already done. So, I'm not sure what you are suggesting, deleting the method  convertToProcessInstanceCustomVars(pi) and using right away the builder's variables?

Reply:
Ok, I understand what you say. Using: _builder().variables(pi.getVariables())_ in the method convertToProcessInstanceCustomVars(pi)  instead of _instance.setVariables(pi.getVariables());_ 
I can change it if you want, I did it like this following the pattern of the other processes mapper.

Reply:
Yes, please. It looks much more refined and easier to read. Thanks!

Reply:
@MarianMacik Done